pandas/tests/sparse/frame/test_analytics.py

```diff
@@ -37,3 +38,16 @@ def test_quantile_multi():
 
     tm.assert_frame_equal(result, dense_expected)
     tm.assert_sp_frame_equal(result, sparse_expected)
+
+
+@pytest.mark.parametrize('func', [np.exp, np.sqrt], ids=lambda x: x.__name__)
```

Comment:
Does the `ids` argument here actually change anything? Tested locally and it was the same with / without that

Reply:
yes it does, as these are ufuncs; updated a bit